DataFormats/HcalRecHit/src/classes_def.xml

```diff
@@ -11,6 +11,12 @@
     <version ClassVersion="11" checksum="1379911758"/>
     <version ClassVersion="10" checksum="1697427050"/>
    </class>
+   <class name="HFQIE10Info" ClassVersion="3">
```

Comment:
Thanks for the pointer.
3 is OK.
I used info from 2015, this twiki was updated recently (2016-02-11).
@Dr15Jones what is the special meaning of version 2?